src/Component/FunctionUI/functionConfigs.ts

```diff
@@ -66,6 +92,18 @@ export const functionConfigs = [
       placeholder: '… enter a number'
     }]
   },
+  {
+    name: 'div',
+    type: 'number',
+    description: 'Returns the division of argument[0] by argument[1]',
```

Comment:
Minor: Might be a little more user friendly
```suggestion
    description: 'Returns the division of the first argument by the second argument',
```